### PR TITLE
Added new keymap for keebio/iris

### PR DESCRIPTION
**Pull request by elliotpatros**
## Description

My personal layout for the keebio/iris. Really simple, only two layers, have been using it for work for several years.

Added keymap.c and config.h to keyboards/keebio/iris/keymaps/emp

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [nearly] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/emp/config.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+// #define NO_ACTION_ONESHOT
+#undef NO_ACTION_ONESHOT
+
+/* disable old style macro handling */
+#define NO_ACTION_MACRO
+
+/* disable calling of action_function() from the fn_actions array */
+// #define NO_ACTION_FUNCTION
+#undef NO_ACTION_FUNCTION
+
```
Comment (zvecr) [inline keyboards/keebio/iris/keymaps/emp/config.h:57]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/emp/keymap.c [hunk 2]
```diff
@@ -0,0 +1,274 @@
+#include QMK_KEYBOARD_H
+
+// Layer definitions
+#define _COLEMAK 0
+#define _FUNCTION 1
+
+// Custom keycodes, implemented later, in function: process_record_user()
+enum custom_keycodes
+{
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/emp/keymap.c:9]:
```suggestion
enum custom_layers {
    _COLEMAK,
    _FUNCTION,
};

// Custom keycodes, implemented later, in function: process_record_user()
enum custom_keycodes {
```

**Review by drashna — CHANGES_REQUESTED** [review]
Also, the formatting bothers me

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**